audio/build.zig

```diff
@@ -4,16 +4,8 @@ const Builder = std.build.Builder;
 const soundio_path = thisDir() ++ "/upstream/soundio";
 
 pub const pkg = std.build.Pkg{
-    .name = "soundio",
-    .source = .{ .path = thisDir() ++ "/soundio/main.zig" },
-};
-
-pub const SoundIoOptions = struct {
```

Comment:
I would be OK with keeping this struct around, just making it private - if you think that's better. I think this way is fine though.